service/src/main/kotlin/fi/espoo/evaka/varda/VardaOrganizerChild.kt

```diff
@@ -79,7 +79,7 @@ private fun createVardaPersonAndChild(
 ): Long {
     val personPayload = getVardaPersonPayload(tx, evakaPersonId, organizerOid)
 
-    val vardaPerson = client.createPerson(personPayload) ?: error("Couldn't create Varda person (nor child)")
+    val vardaPerson = client.createPerson(personPayload) ?: error("VardaUpdate: Couldn't create Varda person $personPayload")
```

Comment:
Is it fine that ssn gets logged?

Reply:
It will be replaced with `REDACTED-SSN` in logs

Reply:
Ssns should be filtered by our log configs.

Reply:
I should refresh the page before answering 😄

Reply:
Ah, nice!